CHANGELOG.md

```diff
@@ -20,6 +20,10 @@
   [J. Cheyo Jimenez](https://github.com/masters3d)
   [#772](https://github.com/realm/SwiftLint/issues/772)
 
+* Fixed: Private unit test rule not scoped to test classes
```

Comment:
missing trailing period and two trailing spaces as defined in [CONTRIBUTING.md](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes).